websites/T/TOD/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "640526616350425100",
+		"name": "itswise"
+	},
+	"service": "TOD",
+	"description": {
+		"en": "TOD is a video-streaming platform which includes a lot of movies, TV shows and matches provided by BeIN.",
+	},
+	"url": [
+		"tod.tv",
+		"www.tod.tv"
+	],
+	"version": "1.0.0",
```

Comment:
There is a new field called  `matches`, which is required for new Presences.
This is an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns.

Reply:
ngl idk if i did the matches correct or nah

Reply:
I think it would be `"*//tod.tv/"`.